packages/cfd/src/Constants/jurisdiction-contents/jurisdiction-svg-contents.ts

```diff
@@ -0,0 +1,86 @@
+import { localize } from '@deriv/translations';
+import { TJurisdictionCardItems } from 'Components/props.types';
+
+export const jurisdictionSvgContents = (): TJurisdictionCardItems => ({
+    is_over_header_available: false,
+    header: localize('St. Vincent & Grenadines'),
+    synthetic_contents: [
+        {
+            key: 'assets',
+            title: `${localize('Assets')}`,
+            description: `${localize('Synthetics, Basket indices and Derived FX')}`,
+            title_indicators: {
+                type: 'displayText',
+                display_text: `${localize('40+')}`,
+                display_text_skin_color: 'red-darker',
+            },
+        },
+        {
+            key: 'leverage',
+            title: `${localize('Leverage')}`,
```

Comment:
U need not wrap the localize function in a template string here. The function will return a localized text that gets binded to the prop